### PR TITLE
Add README for deploying without dhcp

### PR DESCRIPTION
**Pull request by yrobla**

### CODE REVIEW

File: install-ai-operator/README.md [hunk 1]
```diff
@@ -52,3 +52,51 @@ disconnected registry, but you can provide your own using the `disconnected_http
...
+      no-dhcp-interface=baremetal
+      domain=clus2.t5g.lab.eng.bos.redhat.com
+
+      # static host-records
+      address=/apps.test-operator.clus2.t5g.lab.eng.bos.redhat.com/2620:52:0:1310::fb
+      host-record=api.test-operator.clus2.t5g.lab.eng.bos.redhat.com,2620:52:0:1310::fa
+      host-record=test-operator-installer.clus2.t5g.lab.eng.bos.redhat.com,2620:52:0:1310::e8
+      ```
+2. Be sure to inject the fixed static ip to your mirror, so it is resolved to right one.
+
```
Comment (rcarrillocruz) [inline install-ai-operator/README.md:80]: maybe put an example here?

File: install-ai-operator/README.md [hunk 2]
```diff
@@ -52,3 +52,51 @@ disconnected registry, but you can provide your own using the `disconnected_http
       ```
 
 10. Once deployed, you could create clusters using CRDs. This is going to be covered on a different section.
+
+## Deploy with static IPs ##
+
+in order to deploy the cluster, it is mandatory to have a DHCP server and DNS resolution at switch level.
+However, it is possible to deploy with SLAAC IPv6 addresses with some workaround:
```
Comment (rcarrillocruz) [inline install-ai-operator/README.md:59]: did you really mean 'with SLAAC IPV6' ?
Per the readme title, it seems the goal is to make it work without SLAAC/DHCP at all ?

Comment (yrobla) [reply install-ai-operator/README.md:59]: No, we have slaac, is the way we receive the ip addresses. But as there is no dhcp, there is no way to have a fixed ip assignment, give hostnames, or add any nameserver.

File: install-ai-operator/README.md [hunk 3]
```diff
@@ -52,3 +52,51 @@ disconnected registry, but you can provide your own using the `disconnected_http
       ```
 
 10. Once deployed, you could create clusters using CRDs. This is going to be covered on a different section.
+
+## Deploy with static IPs ##
+
+in order to deploy the cluster, it is mandatory to have a DHCP server and DNS resolution at switch level.
```
Comment (rcarrillocruz) [inline install-ai-operator/README.md:58]: What you mean by switch level?

File: install-ai-operator/README.md (hunk 2 above)
Comment (rcarrillocruz) [reply install-ai-operator/README.md:59]: Ah, with SLAAC yes, no with DHCP. I see...

File: install-ai-operator/README.md (hunk 3 above)
Comment (yrobla) [reply install-ai-operator/README.md:58]: That you need to have dhcp and dns service provided by the lab

**Review by rcarrillocruz — APPROVED** [review]

**Pull request merged**